features/storage/kvstore/securestore/mbed_lib.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "SecureStore",
```

Comment:
Styling question : I reviewed few mbed_lib.json files inside our repo. all of them follow lower case with dashes (`device-key` is one example ). Should this be `secure-store` rather?